sip/execution_control/configuration_db/redis/setup.py

```diff
@@ -0,0 +1,15 @@
+"""Setup config file to package configuration database."""
+from setuptools import setup
+
+setup(name='ConfigurationDb',
```

Comment:
Consider changing the `name` to something less generic so we can add it more easily to pip at a later date. 

Suggestion: `skasip-config_db` or `skasip-configuration_db`

Perhaps also consider renaming the code folder from `db_client` to config_db` or `configuration_db` to match the suffix of the package name.

Reply:
I was also thinking of doing this, I have chosen the name skasip-config_db and renamed the folder to config_db.